scripts/make_port_status.py

```diff
@@ -153,6 +158,9 @@ def get_mathlib4_module_commit_info(contents):
             mathlib4_pr=data[node]['mathlib4_pr'],
             source=data[node]['source']
         )
+        _sync_prs = list(sync_prs[data[node]['mathlib4_file']])
+        if _sync_prs:
+            new_status.update(mathlib4_sync_prs=_sync_prs)
```

Comment:
Let's store the labels anyway:
```suggestion
        _sync_prs = []
        for sync_pr_num in sync_prs[data[node]['mathlib4_file']]:
            labels = [dict(name=l.name, color=l.color) for l in prs[sync_pr_num].labels]
            _sync_prs.append(dict(num=sync_pr_num, labels=labels)
        if _sync_prs:
            new_status.update(mathlib4_sync_prs=_sync_prs)
```
I think this might be useful to have available, and I don't think having the yaml grow in size is problematic.